cmd/chisel/cmd_info.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/jessevdk/go-flags"
+	"gopkg.in/yaml.v3"
+
+	"github.com/canonical/chisel/internal/setup"
+)
+
+var shortInfoHelp = "Show information about package slices"
+var longInfoHelp = `
+The info command shows detailed information about package slices.
+
+It accepts a whitespace-separated list of strings. The list can be
+composed of package names, slice names, or a combination of both. The
+default output format is YAML. When multiple package or slice names
+are provided, the output is a list of YAML documents, separated by
+three dashes ("---").
+
+Slice definitions are shown verbatim according to their definition in
+the Chisel releases. For example, globs are not expanded.
```

Comment:
```suggestion
the Chisel Releases. For example, globs are not expanded.
```

I know it's silly, but as nuanced as it might be, we should try to differentiate the chisel-releases from a Chisel (software) release. I don't have a better solution for now (I've actually been using the hyphenized term in other places)

Reply:
Yeah, I use the hyphenized term too. It can be confusing at times.